app/src/test/java/org/mozilla/fenix/ext/BookmarkNodeTest.kt

```diff
@@ -17,12 +17,15 @@ import mozilla.components.concept.storage.BookmarkNodeType
 @Config(application = TestApplication::class)
 class BookmarkNodeTest {
 
+    val bookMarkChild1 = newBookmarkNode("Child 1", 1, null)
```

Comment:
nit: I know you're just copying the existing names, but it would be nice to not capitalize the "m" in bookmark.
```suggestion
    val bookmarkChild1 = newBookmarkNode("Child 1", 1, null)
```